src/components/Filters/Search.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Material } from '@balena/ui-shared-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons/faSearch';
+import { faTimes } from '@fortawesome/free-solid-svg-icons/faTimes';
+
+const { TextField, InputAdornment, IconButton } = Material;
+
+interface SearchProps extends Material.TextFieldProps<'standard'> {
```

Comment:
> Material.TextFieldProps<'standard'>

Haven't seen that, what does it do? 👀

Reply:
this is how Material define TextFieldProps. This extends is related to this comment https://github.com/balena-io-modules/autoui/pull/80#discussion_r1567952439